src/kademlia/dht_tracker.cpp

```diff
@@ -321,10 +321,11 @@ namespace libtorrent { namespace dht {
 		, std::function<void(std::vector<tcp::endpoint> const&)> f)
 	{
 		for (auto& n : m_nodes)
-			n.second.dht.get_peers(ih, f, {}, false);
+			n.second.dht.get_peers(ih, f, {}, {});
```

Comment:
do you think it makes sense to add `::none` to `flags::bitfield_flag`at some point?

Reply:
I kind of like `{}`

Reply:
( https://www.youtube.com/watch?v=NGrnKr9rSz4 )

Reply:
anja, :), looking at it